style.css

```diff
@@ -348,10 +356,401 @@ footer {
   line-height: 24px;
 }
 
-.footer-form > input::placeholder {
+.footer-form input::placeholder {
   font-style: italic;
   color: #6b778c;
   font-size: 15px;
   font-weight: 400;
   line-height: 24px;
 }
+
+.object1b,
+.object2b {
+  display: none;
+}
+
+@media (min-width: 600px) {
+  .first-part {
```

Comment:
[OPTIONAL]
- [ ] Kindly note that for the Desktop breakpoint in the `CSS` file., the value for the `min-width` property should be in the range of `768px` or wider.